tests/lkft/kselftest/skipfile-lkft.yaml

```diff
@@ -0,0 +1,391 @@
+# This skipfile is consumable using skipgen
```

Comment:
Are there any kselftest test cases we need to skip on Altra? If yes, could you add comments to 'reason' field?